src/sage/combinat/combinat.py

```diff
@@ -2918,42 +2918,72 @@ def unshuffle_iterator(a, one=1) -> Iterator:
                (one if sign else - one))
 
 
-def bell_polynomial(n: Integer, k: Integer):
+def bell_polynomial(n: Integer, k=None, ordinary=False):
     r"""
-    Return the Bell Polynomial
+    ``bell_polynomial(n, k)`` return the partial exponential Bell Polynomial
 
     .. MATH::
 
-       B_{n,k}(x_0, x_1, \ldots, x_{n-k}) =
+        B_{n,k}(x_0, x_1, \ldots, x_{n-k}) =
             \sum_{\sum{j_i}=k, \sum{(i+1) j_i}=n}
             \frac{n!}{j_0!j_1!\cdots j_{n-k}!}
             \left(\frac{x_0}{(0+1)!}\right)^{j_0}
             \left(\frac{x_1}{(1+1)!}\right)^{j_1} \cdots
             \left(\frac{x_{n-k}}{(n-k+1)!}\right)^{j_{n-k}}.
 
-    INPUT:
+    If the optional argument ``k`` is omitted,
+    return the complete exponential Bell Polynomial instead
 
-    - ``n`` -- integer
+    .. MATH::
 
-    - ``k`` -- integer
+        B_n(x_0, x_1, \ldots, x_{n-k}) =
+            \sum_{k=0}^n B_{n,k}(x_0, x_1, \ldots, x_{n-k}).
+       
+   Switching the ``ordinary`` optional argument to ``True``,
+   changes the output to the ordinary variant of the Bell polynomials.
 
-    OUTPUT:
+    .. MATH::
 
-    - a polynomial in `n-k+1` variables over `\ZZ`
+        \hat B_{n,k}(x_0, x_1, \ldots, x_{n-k}) =
+            \sum_{\sum{j_i}=k, \sum{(i+1) j_i}=n}
+            \binomial{k}{j_0, j_1, \ldots, j_{n-k}}
+            x_0^{j_0} x_1^{j_1} \cdots x_{n-k}^{j_{n-k}},
 
+   where we have used the multinomial coefficient.
+   The complete version has the same definition as its exponential counterpart.
+   
```

Comment:
```suggestion

   INPUT:
   
   - ``k`` -- (optional) if specified, returns the partial Bell
     polynomial, otherwise returns the complete Bell polynomial
   - ``ordinary`` -- (default: ``False``) if ``True``, returns the
     (partial) ordinary Bell polynomial, otherwise returns
     the (partial) exponential Bell polynomial
```